app/helpers/application_helper.rb

```diff
@@ -1,16 +1,16 @@
 require 'bridge/huboard'
 module ApplicationHelper
   def logged_in?
-    github_authenticated?(:private) || github_authenticated?(:default)
+    github_authenticated?(:default)
   end
   def current_user
-    github_user(:private) || github_user(:default) || OpenStruct.new
+    github_user(:default) || github_user(:private) || github_user(:public) || OpenStruct.new
   end
   def controller? *controller
     (controller.include?(params[:controller]) || controller.include?(params[:action])) ? "nav__btn--active nav__item--current": ''
   end
   def user_token
-    current_user ? current_user.token : nil
+    current_user.token
```

Comment:
Don't need the guard, it was unreachable code. 

the current_user fallback to the an empty OpenStruct that will return nil for token